### PR TITLE
Added Agoo, Racket, Ningle and Ninglex

### PR DESCRIPTION
**Pull request by GertThiel**
Racket is a Scheme dialect, hence a Lisp dialect, including a web framework and server:

https://racket-lang.org/

https://docs.racket-lang.org/web-server/
https://docs.racket-lang.org/web-server/stateless.html

### CODE REVIEW

**Comment by GertThiel** [comment]
Ouch, now this pull request includes the Ningle and Ninglex frameworks as well.

**Comment by GertThiel** [comment]
Weird, on my computer `ninglex.dockerfile` build just fine.

File: .travis.yml [hunk 1]
```diff
@@ -94,6 +94,9 @@ env:
     - "TESTDIR=Java/wizzardo-http"
     - "TESTLANG=JavaScript"
     - "TESTLANG=Kotlin"
+    - "TESTDIR=Lisp/ningle"
+    - "TESTDIR=Lisp/ninglex"
+    - "TESTDIR=Lisp/racket"
```
Comment (nbrady-techempower) [inline .travis.yml]: Could you change these three lines to the single line:

```
      - "TESTLANG=Lisp"
```

to utilize the docker cache and speed up testing. I'll try and do a thorough review of this PR this week. Thanks!

**Comment by GertThiel** [comment]
I updated `.travis.yml` as requested.

On the other hand, this request showed that naming the directory `Lisp` was a bad idea. There is no such thing named Lisp, there are many. And Lisp and Lisp can be very different (and incompatible) things.

`ningle`, `ninglex` and `woo` are all built using the [`woo` HTTP server](https://github.com/fukamachi/woo) compiled with the [Steel Bank Common Lisp](http://www.sbcl.org/) compiler. `- "TESTLANG=Lisp"` should be fine for them. `racket` uses the stateless servlet feature from [Racket](https://racket-lang.org/), a Scheme dialect of the vast Lisp family.

File: frameworks/Scheme/racket/benchmark_config.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                "fortune_url": "/fortunes",
+                "db_url": "/db",
+                "query_url": "/queries?queries=",
+                "update_url": "/updates?queries=",
+                "port": 8080,
+                "approach": "Stripped",
+                "classification": "Micro",
+                "database": "Postgres",
+                "framework": "Racket",
+                "language": "Racket",
```
Comment (nbrady-techempower) [inline frameworks/Scheme/racket/benchmark_config.json]: Could you change this to `Scheme` just to be consistent? If you'd rather distinguish that as language `Racket`, then I'm ok with a `frameworks/Racket/racket` directory.

Comment (GertThiel) [reply frameworks/Scheme/racket/benchmark_config.json]: Maybe we should rename the `Lisp` directory as well to `Common Lisp`!?

File: frameworks/Racket/racket/benchmark_config.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        {
+            "default": {
+                "plaintext_url": "/plaintext",
+                "json_url": "/json",
+                "fortune_url": "/fortunes",
+                "db_url": "/db",
+                "query_url": "/queries?queries=",
+                "update_url": "/updates?queries=",
+                "port": 8080,
+                "approach": "Stripped",
```
Comment (zloster) [inline frameworks/Racket/racket/benchmark_config.json]: Why the Racket implementation is classified as `Stripped`?

File: .travis.yml [hunk 4]
```diff
@@ -103,14 +103,15 @@ env:
...
-    - 'TESTDIR="Ruby/grape Ruby/h2o_mruby Ruby/hanami Ruby/padrino Ruby/rack Ruby/rack-sequel"'
+    - 'TESTDIR="Ruby/agoo Ruby/grape Ruby/h2o_mruby Ruby/hanami Ruby/padrino Ruby/rack Ruby/rack-sequel"'
     - 'TESTDIR="Ruby/rails Ruby/roda-sequel Ruby/sinatra Ruby/sinatra-sequel"'
     - 'TESTDIR="Rust/actix Rust/gotham Rust/hyper Rust/iron Rust/saphir"'
     - 'TESTDIR="Rust/may-minihttp Rust/nickel Rust/rocket"'
     - 'TESTDIR="Rust/rouille Rust/thruster Rust/tokio-minihttp"'
     - 'TESTDIR="Scala/akka-http Scala/blaze Scala/cask Scala/colossus Scala/finagle"'
     - 'TESTDIR="Scala/finatra Scala/finch Scala/http4s"'
     - 'TESTDIR="Scala/play2-scala Scala/youi"'
+    - "TESTLANG=Scheme"
```
Comment (zloster) [inline .travis.yml]: IMO this language is currently unused. [Travis runs doesn't execute anything](https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/609272086).

**Pull request merged**